### ISSUE TITLE
Improving "Run on selected node only"

### ISSUE THREAD

**Issue by ferllings** [ThePacielloGroup/ARC-Toolkit#49]
Some people reported that it no easy to use, and sometime not working.

It should be easy to upgrade this feature to something like:
https://a11y-tools.com/bookmarklets/#isolator

**Comment by douga11y** [comment]
I still frequently see the "Run on selected node only" feature either not work or not appear at all in version 5.6.0. Attaching screen recording showing an example where the option appears, but does not update based on the selected node.

I'm not sure if this matters, but a lot of the pages I test use Javascript frameworks with client-side navigation. Is it possible that dynamically redrawing parts of the DOM breaks this feature?

https://github.com/ThePacielloGroup/ARC-Toolkit/assets/3267932/2aa393e0-aafb-4e33-98fe-012f3a12e4b3

**Comment by ferllings** [comment]
No, that's probably not the case: this feature listen directly for the devtools panel events.
Let's wait for the v5.7.0 that's coming shortly and I'll upgrade that feature from there.
